weldx/geometry.py

```diff
@@ -2330,6 +2330,16 @@ def __post_init__(self):
             if not self.triangles.ndim == 2:
                 raise ValueError("SpatialData triangulation must be a 2d array")
 
+    # def __eq__(self, other):
+    # """Compare to another object."""
+    # if not isinstance(other, SpatialData):
+    #    return False
+    # if not self.coordinates.data == other.coordinates.data:
+    #    return False
+    # if self.triangles and not self.triangles == other.triangles:
+    #    return False
+    # if not self.attributes ==
+
```

Comment:
I guess default `__eq__` doesn't work out of the box because of the arrays?

in that case we should implement a basic `__eq__` function, using `util.compare_nested` for the attributes?

Reply:
I ll try it out